packages/amazonq/test/unit/codewhisperer/util/zipUtil.test.ts

```diff
@@ -173,9 +173,7 @@ describe('zipUtil', function () {
             assert.ok(result.srcPayloadSizeInBytes > 0)
             assert.strictEqual(result.buildPayloadSizeInBytes, 0)
             assert.ok(result.zipFileSizeInBytes > 0)
-            assert.strictEqual(result.lines, 150)
```

Comment:
why are these being removed?

Reply:
I'm not sure about the actual results, but we should remove it since its not related to this PR